app/src/main/res/layout/activity_main.xml

```diff
@@ -58,6 +58,16 @@
 
         <LinearLayout
             android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+            <Button
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="Edit Clipboard Setting"
+                android:onClick="navigateToClipboardModifier"/>
```

Comment:
Could somebody take a look at the margins here? I think we need some, not sure why we don't have a good default.